packages/next/client/image.tsx

```diff
@@ -405,6 +406,19 @@ function akamaiLoader({ root, src, width }: LoaderProps): string {
   return `${root}${normalizeSrc(src)}?imwidth=${width}`
 }
 
+function imageengineLoader({ root, src, width, quality }: LoaderProps): string {
+  const params = ['w_' + width]
+  let paramsString = ''
+  if (quality) {
+    params.push('cmpr_' + (100 - quality))
+  }
+
+  if (params.length) {
+    paramsString = '?imgeng=/' + params.join('/')
```

Comment:
Is `/` intentional here over `&` for the `join`? A comment explaining this would be nice.

Reply:
Yes, the format is like `?imgeng=/w_240/h_100/f_jpg` when used in query params (this is a descendant of a different method of placing these in the beginning of the URI.  I'll add a code comment since it looks strange.